src/Microsoft.OData.Client/Microsoft.OData.Client.csproj

```diff
@@ -449,6 +453,18 @@
     <Compile Include="..\PlatformHelper.cs">
       <Link>PlatformHelper.cs</Link>
     </Compile>
+    <Compile Include="$(EnlistmentRoot)\src\Microsoft.OData.Client\IDataServiceQuerySingle.cs">
+      <Link>IDataServiceQuerySingle.cs</Link>
+    </Compile>
+    <Compile Include="$(EnlistmentRoot)\src\Microsoft.OData.Client\IDataServiceQueryOfT.cs">
+      <Link>IDataServiceQueryOfT.cs</Link>
+    </Compile>
+    <Compile Include="$(EnlistmentRoot)\src\Microsoft.OData.Client\IDataServiceQuery.cs">
+      <Link>IDataServiceQuery.cs</Link>
+    </Compile>
+    <Compile Include="$(EnlistmentRoot)\src\Microsoft.OData.Client\IDataServiceContext.cs">
+      <Link>IDataServiceContext.cs</Link>

```

Comment:
You add the interfaces into "Compile" in this file, why do you add them again and use the "link"?